playbooks/infra-prometheus/setup-prometheus-grafana.yml

```diff
@@ -4,20 +4,29 @@
   become: True
   roles:
     - infra-ansible/roles/config-docker
+  tags:
+    - 'never'
+    - 'install'
 
 - name: Setup node exporters
-  hosts: prometheus_node
+  hosts: prometheus_target
   become: True
   vars:
     provision_state: "started"
   roles:
     - ../../prometheus/generic/setup-node-exporter
+  tags:
+    - 'always'
```

Comment:
This ensures that the play is always run. In case you run the playbook for the first time with --tags="install" it will install docker on all the hosts and then continue with the rest of the playbook. Without always it would do just the first play and only install docker.